### ISSUE TITLE
IOS Android zoom screens could not open but on notification it saying meeting starts

### ISSUE THREAD

**Issue by mohsinmughal1630** [mieszko4/react-native-zoom-us#264]
Please let me know what's i am doing wrong.

that's my zoom screen

const ZoomMeeting = ({ navigation, route }) => {

  const [isInitialized, setIsInitialized] = useState(false);
  const [isMeetingOngoing, setIsMeetingOngoing] = useState(false);

  useEffect(() => {
    setTimeout(async () => {
      try {
        const initializeResult = await ZoomUs.initialize({
          clientKey: 'lOU2OiV8QUW9cK5vTQW6g',
          clientSecret: 'ncviT1NOpziQhBNS5K5Cl6ZXat61IXha',
          // jwtToken : "eyJhbGciOiJIUzI1NiJ9.eyJhdWQiOm51bGwsImlzcyI6InU0RlVSczRoUU1LY2NObzMyVVcwalEiLCJleHAiOjE2NzgwOTk5NTgsImlhdCI6MTY3NzQ5NTE1OX0.Bc4eUBlCHr-ykO33aq28sCXfd-Cel9UEME59N3yoZEg",
          domain: 'zoom.us'
        }, {
          disableShowVideoPreviewWhenJoinMeeting: true,
          enableCustomizedMeetingUI: true
        })

        const isInitialized = await ZoomUs.isInitialized()
        setIsInitialized(isInitialized)
        console.log({ initializeResult }, isInitialized, Platform.OS);

      } catch (e) {
        Alert.alert('Error', 'Could not execute initialize');
        console.error('ERR', JSON.stringify(e));
      }
    }, 1000);
  }, [])

  useEffect(() => {
    const listener = ZoomUs.onMeetingStatusChange(({ event }) => {
      console.log('onMeetingStatusChange', event)
         if (event === 'MEETING_STATUS_CONNECTING') {
          setIsMeetingOngoing(true);
        }

        if (event === 'MEETING_STATUS_DISCONNECTING') {
          // Once it is set it is good to render
          // setIsMeetingOngoing(false);
        }
    })
    const joinListener = ZoomUs.onMeetingJoined(() => {
      console.log('onMeetingJoined')
    })

    return () => {
      listener.remove()
      joinListener.remove()
    }
  }, [])

  const startMeeting = async () => {

    try {
      console.log("eenter")
      const startMeetingResult = await ZoomUs.startMeeting({
        userName: 'John',
        meetingNumber: '0',
        userId: "7O_XNaw7Q2SudlQ3tVpdGg",
        zoomAccessToken: "eyJ0eXAiOiJKV1QiLCJzdiI6IjAwMDAwMSIsInptX3NrbSI6InptX28ybSIsImFsZyI6IkhTMjU2In0.eyJhdWQiOiJjbGllbnRzbSIsInVpZCI6IjdPX1hOYXc3UTJTdWRsUTN0VnBkR2ciLCJpc3MiOiJ3ZWIiLCJzayI6IjAiLCJzdHkiOjEwMCwid2NkIjoidXMwNSIsImNsdCI6MCwiZXhwIjoxNjc3NTA2MTY3LCJpYXQiOjE2Nzc0OTg5NjcsImFpZCI6Ijd3NGM0T0pyUXZXeDhxSnY5VEdFdUEiLCJjaWQiOiIifQ.y0tOZgrGBNt3emSOJvsHwE1RDTftGfxK6yiJUOiJmyI",
        // noMeetingErrorMessage: true, // Set this to be able to show Alert.alert
      }).catch((err) => {
        console.log("printErrrrr", JSON.stringify(err))
      });

      // const leaveMeeting = await ZoomUs.leaveMeeting()
      console.log(startMeetingResult);
    } catch (e) {
      Alert.alert('Error', 'Could not execute startMeeting');
      console.error('ERR', JSON.stringify(e));
    }
  };

  const joinMeeting = async () => {
    try {
      const joinMeetingResult = await ZoomUs.joinMeeting({
        autoConnectAudio: true,
        userName: `Wick ${Platform.OS}`,
        meetingNumber: '81223066404',
        password: 'MW1QMTUxejJzZ3dsKzEvTkZ0SmZkUT09',
        // noMeetingErrorMessage: true, // Set this to be able to show Alert.alert
      });

      console.log({ joinMeetingResult });
    } catch (e) {
      Alert.alert('Error', 'Could not execute joinMeeting');
      console.error('ERR', JSON.stringify(e));
    }
  };

  const dispatch = useDispatch()

  const leaveMeeting = async () => {
    try {
      const leaveMeetingResult = await ZoomUs.leaveMeeting();

      console.log({ leaveMeetingResult });
    } catch (e) {
      Alert.alert('Error', 'Could not execute leaveMeeting');
      console.error('ERR', e);
    }
  };
  return (
    <>
      <View style={styles.container}>
        <Button
          onPress={() => startMeeting()}
          title="Start example meeting"
          disabled={!isInitialized}
        />
        <Text>-------</Text>
        <Button
          onPress={() => joinMeeting()}
          title="Join example meeting"
          disabled={!isInitialized}
        />
      </View>
      {/* {!enableCustomizedMeetingUI && isMeetingOngoing && (
        <CustomViewer leaveMeeting={leaveMeeting} />
      )} */}
    </>
    // <SafeAreaView style={{ flex: 1, backgroundColor: "#fff" }} edges={["left", "right"]}>
    //   <View
    //     style={{
    //       flex: 1,
    //       backgroundColor: "red",
    //       justifyContent: "center",
    //       alignItems: "center"
    //     }}
    //   >
    //     <TouchableOpacity
    //       onPress={async()=>{

    //         startMeeting()
    //         console.log("print -- > ")
    //       }}
    //     >
    //       <View style={{
    //         borderRadius: 5,
    //         height: 50,
    //         paddingHorizontal: 20,
    //         backgroundColor: "gray",
    //         justifyContent: "center",
    //         alignItems: "center"
    //       }}>
    //         <Text style={{
    //           color: "white"
    //         }}>Join the meeting</Text>
    //       </View>
    //     </TouchableOpacity>
    //   </View>
    //   {/* <WebView
    //     //   injectedJavaScript={INJECTEDJAVASCRIPT}
    //       userAgent="Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/78.0.3904.97 Safari/537.36"

    //       source={{ uri: "https://zoom.us/j/95764710114?pwd=aU5hYW9ZNUpBU1dSWUthTE8yQUpLdz09" }}
    //       style={{ width: "100%", height: "100%" }}
    //       renderError={(errorName) => (
    //         <View
    //           style={{
    //             position: "absolute",
    //             left: 0,
    //             right: 0,
    //             height: "100%",
    //             width: "100%",
    //             justifyContent: "center",
    //             alignItems: "center",
    //           }}
    //         >
    //           <Text>Something Went Wrong.</Text>
    //           <Text
    //             style={{ marginTop: 16 }}
    //             onPress={() => {
    //               navigation.goBack();
    //             }}
    //           >
    //             Go Back
    //           </Text>
    //         </View>
    //       )}
    //       setBuiltInZoomControls={true}
    //       setDisplayZoomControls={true}
    //       // renderLoading={() => <ScreenLoader />}
    //       automaticallyAdjustContentInsets={true}
    //       automaticallyAdjustsScrollIndicatorInsets={true}
    //       startInLoadingState={dispatch(startLoader(true))}
    //       pullToRefreshEnabled={true}
    //       onError={(syntheticEvent) => {
    //         const { nativeEvent } = syntheticEvent;
    //         console.log("WebView error: ", nativeEvent);
    //       }}
    //       onLoad={(syntheticEvent) => {
    //         const { nativeEvent } = syntheticEvent;
    //         console.log("WebView onLoad: ", nativeEvent);
    //       }}
    //       onLoadEnd={(syntheticEvent) => {
    //         dispatch(startLoader(false));
    //         // update component to be aware of loading status
    //         const { nativeEvent } = syntheticEvent;
    //         console.log("WebView onLoadEnd: ", nativeEvent);
    //       }}
    //       onLoadStart={(syntheticEvent) => {
    //         // update component to be aware of loading status
    //         const { nativeEvent } = syntheticEvent;
    //         console.log("WebView onLoadStart: ", nativeEvent);
    //       }}
    //       onLoadProgress={({ nativeEvent }) => {
    //         this.loadingProgress = nativeEvent.progress;
    //         console.log("WebView onLoadProgress: ", nativeEvent);
    //       }}
    //       onHttpError={(syntheticEvent) => {
    //         const { nativeEvent } = syntheticEvent;
    //         console.log("WebView received error status code: ", nativeEvent.statusCode);
    //       }}
    //       onNavigationStateChange={(navState) => {
    //         // Keep track of going back navigation within component
    //         console.log("WebView onNavigationStateChange: ", navState);
    //       }}
    //     /> */}
    // </SafeAreaView>
  )
}

**Comment by mohsinmughal1630** [comment]
JNI DETECTED ERROR IN APPLICATION: JNI CallIntMethodV called with pending exception java.lang.UnsatisfiedLinkError: dlopen failed: cannot locate symbol “__emutls_get_address”

**Comment by mieszko4** [comment]
It is hard to follow the code without indentation and syntax highlight...

Anyway, `JNI DETECTED ERROR IN APPLICATION` sounds like you have some native conflicting lib included . Which version of the lib do you use? Have you tried with [test app](https://github.com/mieszko4/react-native-zoom-us-test) ?

**Comment by mohsinmughal1630** [comment]
I use all these 6.16.5, 6.15.1 and 6.14.1 but getting same error.

Secondly yes i try your test app and in this i get same error.

**Comment by mieszko4** [comment]
Maybe it is related to specific Android version that you use? Do you test on emulator or real device?

**Comment by mohsinmughal1630** [comment]
I check this in multiple android versions
Android 9, 10, 11
Also i checked on emulator and real devices as well
I am using arm mac x86

**Comment by mieszko4** [comment]
Try with 6.18.0 - it uses Android ZoomSDK 5.13.10.

**Comment by lamlengend98** [comment]
@mieszko4 updated to 6.18.2 and still occured that

**Comment by mieszko4** [comment]
Please try with react-native-zoom-us@6.20.0 (with updated iOS ZoomSDK to 5.14.11.8690 and updated Android ZoomSDK to 5.16.2.16555)
